src/main/java/sg/ncl/testbed_interface/UtilizationUtil.java

```diff
@@ -0,0 +1,23 @@
+package sg.ncl.testbed_interface;
+
+import java.time.YearMonth;
+import java.time.Month;
+
+public class UtilizationUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)